wwwroot/index.html

```diff
@@ -1,198 +1,385 @@
+                            <p class="ms-5">Coinjoin with any amount, from 5,000 sats to 40,000 BTC and anything in between. WabiSabi is the coinjoin protocol under the hood that makes it work. </p>
+                            <h3 class="title-small position-relative ms-5 mt-5"> <img src="img/icon6.svg" class="icon-pull-left"/> AFFORDABLE </h3>
+                            <p class="ms-5">WabiSabi is a next generation coinjoin protocol, designed with blockspace efficiency as a priority so you save on mining fees.</p>
+                        </div>
                     </div>
                 </div>
             </div>
         </section>
-        <section class="home p-100 green-bordered-section" id="download">
+        <section id="coinjoin" class="coinjoin-section">
             <div class="container">
-                <div class="row align-items-center mt-5 mt-md-0">
-                    <div class="col-xl-6 offset-xl-3">
-                        <div class="text-center">
-                            <h2 class="title title-lined">DOWNLOAD<br>WASABI WALLET</h2>
+                <div class="row">
+                    <div class="col-lg-3 offset-lg-1">
+                        <h2 class="title">A coinjoin is...</h2>
+                    </div>
+                    <div class="col-lg-4 mt-5 mt-lg-0">
+                        <p> A collaborative bitcoin transaction. In a coinjoin, many different users coordinate the creation of one single transaction with inputs and outputs from different users. </p>
+                        <p> A coinjoin transaction has many standard outputs of the same amounts, which makes it very hard to track them back to their initial inputs. </p>
+                        <p> Coinjoins allow users to conceal the history of their coins from the public to make their UTXOs fungible on the public Bitcoin network. </p>
+                    </div>
+                    <div class="col-lg-2 offset-lg-1 position-relative"> <img src="img/screen_shot2.png" class="pull-img pull-img-to-right" /> </div>
+                </div>
+            </div>
+        </section>
+        <section>
+            <div class="container">
+                <div class="row align-items-end">
+                    <div class="col-lg-8 offset-lg-1">
+                        <div class="row">
+                            <div class="col-lg-6">
+                                <h2 class="title">Mining fees and coordinator fees</h2>
+                            </div>
+                            <div class="col-lg-6">
+                                <p>Every individual user is responsible for paying their own mining fee for the block space they utilize. This arrangement guarantees that the expenses associated with transactions are equitably allocated among all users, ensuring fairness in the distribution of costs.</p>
```

Comment:
lettings users pay for their blockspace, a.k.a. everyone responsible for their own mining fee is also a coordinator policy